family2family/build.gradle

```diff
@@ -11,13 +10,18 @@ buildscript {
     }
 }
 
+// Update appropriately for releases
+version = '1.0.0-SNAPSHOT'
+
 apply plugin: 'java'
 apply plugin: 'idea'
 apply plugin: 'org.springframework.boot'
 
 sourceCompatibility = 1.8
 targetCompatibility = 1.8
 
+jar.archiveName = "family2family.${jar.extension}"
```

Comment:
Any opinions?  Without this line, the produced jar will be `family2family-1.0.0-SNAPSHOT.jar`.  Not sure if the deployment process literally expects `family2family.jar` or what.